setup.py

```diff
@@ -7,7 +7,7 @@
     author='Trey V. Wenger',
     author_email='tvwenger@gmail.com',
     packages=['maxfield'],
-    install_requires=['numpy', 'networkx', 'scipy', 'ortools', 'protobuf==3.19.4',
+    install_requires=['numpy', 'networkx', 'scipy', 'ortools', 'protobuf==4.21.5',
```

Comment:
Ortools now wants protobuf >= 4.25.3: https://github.com/google/or-tools/blob/stable/ortools/python/setup.py.in

Maybe we should use `>=` or `~=` instead of `==`?
```suggestion
    install_requires=['numpy', 'networkx', 'scipy', 'ortools', 'protobuf~=4.21.5',
```